data/assets/scene/solarsystem/dwarf_planets/pluto/charon/charon.asset

```diff
@@ -27,19 +27,14 @@ local Charon = {
     SegmentsPerPatch = 64,
     Layers = {},
     Labels = {
-      Enable = false,
+      Enabled = false,
       FileName = labelsPath .. "/charon.labels",
       LabelAlignmentOption = "Horizontally", -- or Circularly
       LabelsFontSize = 40.0,
       LabelsSize = 8.0,
```

Comment:
No, no need. The globe labels do not vary as much in size as the other label types, so here it wakes sense to just use the default values. I even removed the Min Max Size for all the globes that had them assigned, since they were simply just set to the default values for all globes. 

Also reduced the default max value from 1500 px to 1000 px. Could not find a use case where this made a difference and it made the slider more interactible (in most cases we don't get a visible change unless setting the max to something below 150-ish, at least on monitors). Realizing not that it might make a difference on more high-res displays, such as the dome? If that is the case I should probably change it back to 1500